packages/web/src/components/common/wallet-connector-menu/WalletConnectorMenu.styles.ts

```diff
@@ -3,11 +3,19 @@ import styled from "@emotion/styled";
 import { media } from "@styles/media";
 import mixins from "@styles/mixins";
 import { Z_INDEX } from "@styles/zIndex";
+import { CSSProperties } from "react";
 
-export const WalletConnectorMenuWrapper = styled.div`
+export interface WalletMenuProps {
+  width: CSSProperties["width"];
+}
+
+export const WalletConnectorMenuWrapper = styled.div<WalletMenuProps>`
   position: absolute;
   width: 280px;
   top: 54px;
+  right: ${({ width }) => {
+    return width && width < 1521 && "0px";
```

Comment:
The type of width is either string or number. If it is a string, it cannot be compared to 1521.

Reply:
correct.